pepper-apis/src/main/java/org/broadinstitute/ddp/route/ReceiveDsmNotificationRoute.java

```diff
@@ -79,6 +87,11 @@ public Object handle(Request request, Response response, DsmNotificationPayload
                 testResult = parseTestResult(response, payload);
             }
 
+            if (eventType == DsmNotificationEventType.TESTBOSTON_SENT) {
```

Comment:
How specific is this to tbos, and should we consider using a lookup here to determine that it's for tbos?  My concern here is the potential for a patten of study-specific events over the long haul.

Reply:
`TESTBOSTON` is a kit type so this is specific to kit types rather than studies. We have other kit types like `SALIVA` and `BLOOD`. We do have the `SALIVA_SENT` event, but no `BLOOD_SENT` event. We can add support for those other kit types when they get used for scheduling longitudinal kits.